### PR TITLE
revise dartdoc integration

### PR DESCRIPTION
**Pull request by devoncarew**
- tweaks to the UI for the documentation view
- update the url we built to work with the new dartdoc

@lukechurch

### CODE REVIEW

**Comment by devoncarew** [comment]
The build failure looks related to bower getting the polymer components. Investigating...

File: lib/documentation.dart [hunk 1]
```diff
@@ -175,16 +176,20 @@ ${libraryName == null ? '' : apiLink }\n\n''';
...
-        if (enclosingClassName == null) {
-          apiLink.write(memberName);
-        } else {
-          apiLink.write(".${enclosingClassName}${memberName}");
-        }
-        return '[Open external docs](${apiLink})';
+        libraryName = libraryName.replaceAll(':', '-');
+        apiLink.write('https://api.dartlang.org/stable/${libraryName}/${libraryName}-library.html');
+
+        // if (enclosingClassName == null && memberName == null) {
```
Comment (lukechurch) [inline lib/documentation.dart]: This isn't needed any more because of the new structure? Can we just delete it?

**Comment by lukechurch** [comment]
lgtm

File: lib/documentation.dart (hunk 1 above)
Comment (devoncarew) [inline lib/documentation.dart]: sgtm

**Pull request merged**